lib/publiccloud/aws.pm

```diff
@@ -68,4 +69,11 @@ sub init {
     }
 }
 
+sub get_user_id {
```

Comment:
so in my understanding this function will be dropped ? ( it contains of 2 lines and one goes to prepare_tools module and another in to `publiccloud::aws` init code )

Reply:
done